operations/iuf/workflows/slingshot_management_network_switch_updates.md

```diff
@@ -0,0 +1,31 @@
+# Perform Slingshot switch and management network switch firmware updates
+
+This section updates the Slingshot switch and management network switch firmware updates.
+
+- [1. Perform Slingshot switch firmware updates](#1-perform-slingshot-switch-firmware-updates)
+- [2. Perform management network switch firmware updates](#2-perform-management-network-switch-firmware-updates)
+
+## 1. Perform Slingshot switch firmware updates
+
+**`NOTE`** This subsection is optional and can be skipped if upgrading only CSM through IUF.
+
+Instructions to perform Slingshot switch firmware updates are provided in the "Upgrade HPE Slingshot switch firmware in a CSM environment" section of the _HPE Slingshot Installation Guide for CSM_.
+
+Once this step has completed:
+
+- Slingshot switch firmware has been updated
+
+## 2. Perform management network switch firmware updates
+
+**`NOTE`** This section is optional and can be skipped or deferred unless network configuration that requires updated firmware is being applied to the system.
+
+Management network switch firmware is shipped in the HPC Firmware Pack (HFP) product tarball.
```

Comment:
Is this true for non-HPE branded switches (e.g. Dell, Mellanox)?

Reply:
It does appear that the Dell and Mellanox firmware is no longer in HFP however the recommended firmware revisions for these switches have not changed since CSM 1.2

Reply:
No changes are needed since these switches have not changed since CSM 1.2.